jasmine/private/junit_reporter.js

```diff
@@ -0,0 +1,13 @@
+const reporters = require("jasmine-reporters");
+const path = require("path");
+
+const xmlOutputFile = process.env.XML_OUTPUT_FILE;
+
+jasmine.getEnv().addReporter(
```

Comment:
I assume `jasmine` is a global but is it possible to `require()` it instead? 🤷

Reply:
this is the recommended pattern by jasmine. if a require jasmine, it'll give me the library itself not the instance of it. thus I can't do getEnv() on an uninitialized instance.

Reply:
I'll make sure that this is the case though.

Reply:
yeap. that's the case. i get `TypeError: require(...).getEnv is not a function`